easybuild/easyconfigs/t/Trinity/Trinity-2.0.6-goolf-1.4.10.eb

```diff
@@ -0,0 +1,47 @@
+# This file is an EasyBuild reciPY as per https://github.com/hpcugent/easybuild
+# Author: Pablo Escobar Lopez
+# Swiss Institute of Bioinformatics
+# Biozentrum - University of Basel
+
+easyblock = 'ConfigureMake'
+
+name = 'Trinity'
+version = '2.0.6'
+
+homepage = 'http://trinityrnaseq.github.io/'
+description = """ Trinity assembles transcript sequences from Illumina RNA-Seq data. """
+
+toolchain = {'name': 'goolf', 'version': '1.4.10'}
+
+source_urls = ['https://github.com/trinityrnaseq/trinityrnaseq/archive/']
+sources = ['v%(version)s.tar.gz']
+
+unpack_options = '--strip-components=1'
+
+java = 'Java'
+javaver = '1.7.0_21'
```

Comment:
@fgeorgatos @fotis I would not go 1.8.x. Most of the bioinfo packages are tested only with 1.7.x or even 1.6.x 

if you want to get 1.8.x here to get it merged it's fine for me but I will modify it for my local installation :)